### ISSUE TITLE
add basic tensorboard summaries

### ISSUE THREAD

**Issue by jacobsn** [UkyVision/bird-audio-detection#11]
A few useful things:
- loss
- learning rate
- mini-batch accuracy (training data)
- (maybe) mini-batch accuracy (test data)

This will allow us to more easily monitor training progress and compare different methods.  Output to logs/{run_name}/ so we can see them all together.
